### PR TITLE
Aymenby caption

### PR DESCRIPTION
**Pull request by AYMENCARLOS**
customize the caption text

### CODE REVIEW

File: _posts/0000-01-09-AYMENCARLOS.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![octofez](https://octodex.github.com/images/octofez.png)
 {: .center}
 
-Caption here
+I git it
```
Comment (flaviaportinari) [inline _posts/0000-01-09-AYMENCARLOS.md:9]: lol

**Review by flaviaportinari — APPROVED** [review]
ok

File: _posts/0000-01-09-AYMENCARLOS.md (hunk 1 above)
Comment (anescum) [inline _posts/0000-01-09-AYMENCARLOS.md:9]: I love this change!

**Review by anescum — APPROVED** [review]
And now I approve it.

File: _posts/0000-01-09-AYMENCARLOS.md (hunk 1 above)
Comment (AYMENCARLOS) [reply _posts/0000-01-09-AYMENCARLOS.md:9]: thx

**Comment by AYMENCARLOS** [comment]
this is my message for yoouuu

**Comment by AYMENCARLOS** [comment]
Leave a comment

**Pull request merged**